### ISSUE TITLE
[Feature]: Different Models for Launch Pad and Buggy Fueling Pad

### ISSUE THREAD

**Issue by TheMCSpacePro** [TeamGalacticraft/Galacticraft-Legacy#144]
### Searched For Existing Request?

Yes I Have

### Feature Description

Please note that by "pad model", I'm referring to the completed 9x9 pad. Also, these new pad models would be toggable in the config, if turned off, the pads would just use their current GC4 model. 

My main thing with the current models is that the middle piece that sticks up is all we can modify. It looks even worse on the buggy fueling pad since the "full" texture only spans that middle piece. Hence this suggestion, looking into Galacticraft's past and future to make it nicer.

The buggy fueling pad's model would become completely flat and the texture would span the entire pad. In layman's terms, how it looked in GC3 and piror. This pad model would be ON by default.

The launch pad's model would be similar to it's model in GC5, simply using a texture better suited for GC4. The texture would still go across the entire 9x9 pad just as in GC5 as well. Due to resource packs, this would be OFF by default.

**Comment by ROMVoid95** [comment]
Will not be changing models in Legacy
